extensions/amp-video/0.1/validator-amp-video.protoascii

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
```

Comment:
we still need the "script" section you had in the previous PR. with `unique_warning: true` changed to `unique: true` as @honeybadgerdontcare suggested.

(The script section is about validation rules on the `<script src=..amp-video-0.1->` itself, something that did no exist before but now is needed since amp-video is its own extension now)